documentation/src/main/asciidoc/topics/proc_exporting_metrics_state_transfer.adoc

```diff
@@ -0,0 +1,33 @@
+[id='displaying-metrics-state-transfer_{context}']
+= Exporting metrics during a state transfer operation
+
+You can export time metrics for clustered caches that {brandname} redistributes across nodes. 
+
+A state transfer operation occurs when a clustered cache topology changes, such as a node joining or leaving a cluster.
+During a state transfer operation, {brandname} exports metrics from each cache, so that you can determine a cache's status.
+A state transfer exposes attributes as properties, so that {brandname} can export metrics from each cache.
+
+[NOTE]
+====
+You cannot perform a state transfer operation in invalidation mode. 
+====
+
+{brandname} generates time metrics that are compatible with the REST API and the JMX API. 
+
+.Prerequisites
+* Configure {brandname} metrics.
+* Initiate a state transfer operation by changing your clustered cache topology. 
+* Enable metrics for your cache type, such as embedded cache or remote cache.
```

Comment:
Where do you see that embedded or remote are cache types?
We should be consistent and use the same terminology and the same applies to statistics and metrics. Go with the terminology that is used

Reply:
"Cache types" is not a specific term I'm trying to specify. I am speaking in a generalized tone to refer to embedded caches and remote caches. 

My only suggestion is:

> Enable metrics for either your embedded cache or yoru remote cache.

WDYT? Originally, I felt I needed to add "type" to avoid a misinterpretation that embedded and remote caches were metrics.

Reply:
Enable metrics for your clustered cache?

Reply:
Clustered caches might be mislead because the following document section explains clustered caches as replicated, distributed, and so on:

[Clustered caches](https://infinispan.org/docs/stable/titles/configuring/configuring.html#clustered-caches)

Reply:
right, those are cache modes https://infinispan.org/docs/stable/titles/configuring/configuring.html#cache-mode-comparison_caches 
But this is fine